src/js/views/patients/worklist/worklist_views.js

```diff
@@ -182,6 +182,11 @@ const FlowItemView = View.extend({
 
     this.showChildView('owner', ownerComponent);
   },
+  setProgress() {
+    const progress = this.model.get('_progress');
+
+    this.$el.find('.worklist-flow__progress').attr({ value: progress.complete, max: progress.total });
```

Comment:
if these values aren't going to change we might as well put them in the template.. for that matter they should always be put in the template, and this function in the other views should just be called `onChangeFlowState`